### PR TITLE
Feature/484 funding prompt flow

### PR DESCRIPTION
**Pull request by achillegley**

### CODE REVIEW

File: src/components/Admin/FundDepositCard.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<template>
+  <q-card>
+    <q-card-section class="q-pt-none">
+      <q-form @submit="onSubmit">
+        <!-- <q-input hide-hint label="Winner wallet address" maxlength="80" required v-model="_walletAddress" disable /> -->
```
Comment (marclupanc) [inline src/components/Admin/FundDepositCard.vue:5]: Please remove all the commented code

File: .gitignore [hunk 2]
```diff
@@ -41,3 +41,7 @@ coverage/
 conf-example
 functions/node_modules
 package-lock.json
+
+package-lock.json
+package.json
+quasar.config.js
```
Comment (marclupanc) [inline .gitignore:47]: Why is it necessary to have this in gitignore?

File: src/components/Advertiser/AdvertiserTable.vue [hunk 3]
```diff
@@ -384,7 +384,7 @@ async function _claimPayment(advertise, currentAmountSpent) {
   $q.loading.show()
   const result = await claimPayment({ campaignCode: advertise.campaignCode, currentAmounSpentInMatic: currentAmountSpent })
   if (result.status.includes('success')) {
-    console.log('the result claimPayment result ====', result)
+    //console.log('the result claimPayment result ====', result)
```
Comment (marclupanc) [inline src/components/Advertiser/AdvertiserTable.vue:387]: Please remove all the commented code

File: src/components/Admin/ManagePromptsEntries.vue [hunk 4]
```diff
@@ -248,4 +282,21 @@ function handleDeleteEntry(entryId, promptId) {
 
   promptStore.fetchPrompts()
 }
+
+//proceed deposit funds.
+async function onProceedDepositFundDialog(props) {
+  //let's check if the entry already have valid payment..
+  //console.log('the current wallet address ', currentWalletAddress)
```
Comment (marclupanc) [inline src/components/Admin/ManagePromptsEntries.vue:289]: Please remove all the commented code

**Review by marclupanc — CHANGES_REQUESTED** [review]
Does this PR has all the changes from - #543 ?

**Comment by achillegley** [comment]
> Does this PR has all the changes from - #543 ?

yes it has all my bad.

**Review by marclupanc — COMMENTED** [review]
Maybe we should disallow admins to delete prompts that currently have an entry with unclaimed payment? Currently if a user removes an prompt and user din't claim the payment will the user be able to claim it later?

**Review by marclupanc — APPROVED** [review]

**Pull request merged**